bundler/spec/install/gemfile/platform_spec.rb

```diff
@@ -550,7 +550,7 @@
     gemfile <<-G
       source "#{file_uri_for(gem_repo1)}"
 
-      gem "rack", :platform => [:windows, :mingw, :mswin, :x64_mingw, :jruby]
+      gem "rack", :platform => [:windows, :mswin, :mswin64, :mingw, :x64_mingw, :jruby]
```

Comment:
What's this change for?

Reply:
`:mswin64` looked like it was unintentionally missing here, as all other windows platforms were there.

Reply:
Hmm I think these should still be left in the spec, as we want to ensure they all still work even though they are considered deprecated.

Reply:
But the spec seems completely unrelated to Windows

> it "does not print a warning when a dependency is unused on a platform different from the current one"

That's why I'd either leave only `:windows` there to somehow link the change to this PR, or leave this spec completely untouched.

Anyways, it's no big deal really.

Reply:
Meh, lets do the way I have in this PR. Err on the side of more coverage.

Reply:
I don't see this as more coverage, but I'm not going to fight over this. Approving.